app/components/address.js

```diff
@@ -0,0 +1,8 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+export default class AddressComponent extends Component {
+    @tracked name = 'Mr. Walter Brown';
+    @tracked location = 'Flat29 North-London UB24WLL';
```

Comment:
have postcode as something like this PO16 7GZ

Reply:
changed